exporters/committime/collector_base.py

```diff
@@ -182,7 +182,11 @@ def get_metric_from_build(self, build, app, namespace, repo_url):
                     repo_url = self._get_repo_from_build_config(build)
 
             metric.repo_url = repo_url
-            commit_sha = build.spec.revision.git.commit
+
+            if build.spec.revision is None:
+                commit_sha = self._get_revision_from_build(build)
+            else:
```

Comment:
Should we add logic to get commit_sha if one was not resolved from the self._get_revision_from_build() func and not always when the build.spec.revision is not None?